qml/components/ChumDetailItem.qml

```diff
@@ -7,10 +7,11 @@ Label {
 
   anchors {
     left: parent.left
-    leftMargin: Theme.horizontalPageMargin
     right: parent.right
     rightMargin: Theme.horizontalPageMargin
   }
+  topPadding: Theme.paddingSmall
+  bottomPadding: Theme.paddingSmall
   color: Theme.highlightColor
   wrapMode: Text.WrapAtWordBoundaryOrAnywhere
   font.pixelSize: Theme.fontSizeExtraSmall
```

Comment:
In similar note as for description text, try to bump this to `Theme.fontSizeSmall`. I would argue that it is bad design to have to use extrasmall fonts. Exception would be numbers with forks/stars or number of comments as numbers are somewhat larger than small chars.